### ISSUE TITLE
Loinc 27088-4:  "Folate [Presence] in Blood"

### ISSUE THREAD

**Issue by kingmanzhang** [obophenotype/human-phenotype-ontology#4019]
Suggest creating a new child term of HP:0040087 [Abnormality of folate in blood] for Loinc 27088-4 [Folate [Presence] in Blood]
New term label: nothing
New term comment (if any): it is a child of Folate deficiency--strange relationship
Your biocurator ID for loinc2hpo (if desired): JGM:azhang

**Comment by pnrobinson** [comment]
Folate in blood is not an abnormal finding. I need to research this but a quick search shows that the normal range is 
The mean plasma folate  (normal value, 2.7-34.0 ng/mL) 
I have never heard of an increased value being clinically important. 
Reduced folate is an important phenotype however.

**Comment by balhoff** [comment]
@kingmanzhang the logical definition for 'Abnormality of folate in blood' specifies 'decreased amount', and the only difference in definition compared to 'Folate deficiency' is that it specifies 'part_of some blood'. It seems like the wrong quality is being used, and should probably just be 'amount'. Also, should 'Folate deficiency' specify 'part_of some blood' to be consistent with 'Abnormality of folate in blood'?

**Comment by nicolevasilevsky** [comment]
Need to merge HP_0040087 with parent class HP_0100507

**Comment by nicolevasilevsky** [comment]
to merge- add ID as alt ID and label as exact syn and then obosolete class

**Comment by nicolevasilevsky** [comment]
@pnrobinson and @kingmanzhang 
I think that HP_0040087 should be renamed to 'Folate deficiency in blood' (I don't think it shouldn't be merged with it's parent class, as noted above.)

Shouldn't the logical def for _HP_0012336 'Reduced cerebrospinal fluid 5-methyltetrahydrofolate concentration'_ be:
'has part' some 
('decreased amount' and ('inheres in' some 
(5-methyltetrahydrofolate and ('part of' some 'cerebrospinal fluid'))) and ('has modifier' some abnormal))
?

Right now, it has the quality _amount_

**Comment by pnrobinson** [comment]
@nicolevasilevsky I updated the definition to 'decreased amount', thanks for picking up this error!

**Comment by pnrobinson** [comment]
I am checking the annotations for the folate terms to see if we can merge them.
Abnormality of folate in blood: https://hpo.jax.org/app/browse/term/HP:0040087
```
$ grep 'HP:0040087' phenotype.hpoa 
OMIM:275350	#275350 TRANSCOBALAMIN II DEFICIENCY;;TC II DEFICIENCY;;TCN2 DEFICIENCY	NOT	HP:0040087	OMIM:275350	TAS	PHPO:skoehler[2014-11-27]
OMIM:613839	#613839 MEGALOBLASTIC ANEMIA DUE TO DIHYDROFOLATE REDUCTASE DEFICIENCY;;DHFR DEFICIENCY	NOT	HP:0040087	OMIM:613839	TAS					P	HPO:skoehler[2014-11-27]
```

and Folate deficiency https://hpo.jax.org/app/browse/term/HP:0100507
```
$ grep 'HP:0040087' phenotype.hpoa 
OMIM:275350	#275350 TRANSCOBALAMIN II DEFICIENCY;;TC II DEFICIENCY;;TCN2 DEFICIENCY	NOT	HP:0040087	OMIM:275350	TAS	PHPO:skoehler[2014-11-27]
OMIM:613839	#613839 MEGALOBLASTIC ANEMIA DUE TO DIHYDROFOLATE REDUCTASE DEFICIENCY;;DHFR DEFICIENCY	NOT	HP:0040087	OMIM:613839	TAS					P	HPO:skoehler[2014-11-27]
[~/GIT/hpo-annotation-data/rare-diseases/misc_2018] $ grep 'HP:0100507' phenotype.hpoa 
OMIM:219721	219721 CYSTIC FIBROSIS WITH HELICOBACTER PYLORI GASTRITIS, MEGALOBLASTICANEMIA, AND MENTAL RETARDATION		HP:0100507	OMIM:219721	TAS					P	HPO:skoehler[2013-09-17]
OMIM:212750	#212750 CELIAC DISEASE; CD;;CELIAC SPRUE;;GLUTEN-SENSITIVE ENTEROPATHY; GSECELIAC DISEASE, SUSCEPTIBILITY TO, 1, INCLUDED; CELIAC1, INCLUDED		HP:0100507	OMIM:212750	TAS					P	HPO:skoehler[2013-09-17]
```

**Comment by pnrobinson** [comment]
OMIM:275350 and OMIM:613839	have Normal serum folate (NOT HP:0040087 in our annotation)
OMIM:219721	and OMIM:212750	have folate deficiency (HP:0100507)

**Comment by pnrobinson** [comment]
We should probably also have a term for increased serum folate, because (for instance), Pernicious anemia may cause high serum folate levels, due to impaired utilization of folate.

**Comment by pnrobinson** [comment]
done revising these terms.
